circle/circle.go

```diff
@@ -18,6 +18,15 @@ Square: 201.06192982974676
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
 	// ++ Your code here! ++
+	fmt.Println("Enter the radius:")
+	var rds float64
+	fmt.Scan(&rds)
+	var Pi float64 = 3.14159265358979323846264338327950288419716939937510582097494459
+	var prm = 2 * (Pi * rds)
```

Comment:
optional: Please consider using [short variable declaration](https://golang.org/ref/spec#Short_variable_declarations) operator `:=` instead. It would look like this:
```go
p := 2 * math.Pi * r
a := math.Pi * r * r
```